src/fetch_data.py

```diff
@@ -0,0 +1,38 @@
+import requests
+import os
+from src.logs.logger import DataProcessingLogger
+
+
+
+
+def main():
+    logger = DataProcessingLogger("src/logs/indexer_data/indexer.log")
+
+    indexer_uri = os.getenv("INDEXER_URI")
+    raw = requests.post(indexer_uri+"/graphql", json = {
+        "operationName": "getTransactions",
```

Comment:
We could move this constant graphql strings to some different file

Reply:
i created basic IndexerService where all logic will be (in future)